ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/SigningEeaSendTransactionIntegrationTest.java

```diff
@@ -72,13 +72,6 @@ public void invalidParamsResponseWhenSenderAddressIsTooLong() {
         response.ethSigner(SIGNING_FROM_IS_NOT_AN_UNLOCKED_ACCOUNT));
   }
 
```

Comment:
do you want to keep this but prove it IS valid input?

Reply:
good idea. I've added that back now together with the missing prefix tests